tests/test_services_zeropush.py

```diff
@@ -117,7 +117,7 @@ def request_callback(request):
         send = zeropush.send_push_notification(devices, 'This is the message',
                                                sound="annoyingSound.mp3",
                                                badge_number=1,
-                                               payload={
+                                               custom={
```

Comment:
Generally `payload` is what we are actually sending to notification service. In this case it the dict that is generated when combining all args/kwargs.